src/components/single_use_components/ProfileListOptions.js

```diff
@@ -10,8 +10,11 @@ import {
 } from "../../helpers/modalFactory";
 import { handleSignOut } from "../../helpers/signOut";
 import { MOCK_USER } from "../../constants/MockUser";
+import AlgoquantApiContext from "../../constants/ApiContext";
 
 export default function ProfileListOptions(props, { navigation }) {
+  const algoquantApi = useContext(AlgoquantApiContext);
+  const [alpacaConnection, setAlpacaConnection] = useState(false);
   // Set the information for the corresponding modal
   // (This will be extracted into a separate file containing all functions for setting information for different types of modals)
```

Comment:
Is this still intended to be extracted to a separate file ?

Reply:
what do you mean by this? this is needed in this component so we can call the API get user again, instead of passing it from TotalBalance.js

Reply:
The comment says : 

 // (This will be extracted into a separate file containing all functions for setting information for different types of modals)

Reply:
thats comment is below the lines of code u highlighted for this

Reply:
So did Will write that comment ?

Reply:
yea lol @WillOrtiz7 wanna address this

Reply:
> yea lol @WillOrtiz7 wanna address this

I forgot to remove that comment. The stuff that needed to be moved to a different file was already moved